src/app/user-details/user-details-content.tsx

```
@@ -284,13 +289,31 @@ export function UserDetailsContent() {
+                            transform hover:scale-105 min-w-[140px]"
+                          disabled={bundle.remainingUses <= 0 || isNavigatingToPayment}
+                          size="default"
                         >
-                          {bundle.remainingUses > 0 
-                            ? (language === "en" ? "Use" : "Usar")
-                            : (language === "en" ? "Empty" : "Vacío")}
+                          {isNavigatingToPayment ? (
+                            <>
+                              <motion.div
+                                animate={{ rotate: 360 }}
+                                transition={{ duration: 1, repeat: Infinity, ease: "linear" }}
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)